### PR TITLE
docs: Added CONTRIBUTIONS.md

### LINKED ISSUE #822: Add documentation about contribution and finding issues
Add documentation on how to contribute by finding issues

The first step should be to check `core` tags, and `high-priority` ones

Then `unimplemented` issues should be checked which got closed due to lack of attention but still need to be implemented.

Then `deferred` issues may be valid now to take up

If no such criteria meets, only then move on to create new issues during an issue hunt. Of course issues about serious bugs and new ideas about features are always welcome.

Above paragraph is a gist of what is to be communicated

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #822 

Changes: A contributions.md section was added and procedure for solving issues was written

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@iamareebjamal Kindly review

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: owners -> maintainers

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: link COC

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Issues Process
+
+1. First step while searching for issues is to check the ones with `core` and `high-priority`.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: ... core and high priority labels

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+2. Then `unimplemented` issues should be checked which got closed due to lack of attention but still need to be implemented.
+3. Then `deferred` issues may be valid now to take up.
+4. If no such criteria meets, only then move on to create new issues during an issue hunt. Of course issues about serious bugs and new ideas about features are always welcome.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: Remove `only`

Comment (iamareebjamal) [inline CONTRIBUTING.md]: Add create new issues using issue buttons (link)(README)here

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: Change to `Ensure that the tests are passing locally by running ./gradlew build and  there are no checkstyle or PMD issues`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+2. Check that there are no conflicts and your request passes Travis build. Check the log of the pass test if it fails the build.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: Rephrase second line

**Comment by sridharjajoo** [comment]
@iamareebjamal I am closing this PR and will be sending other P.R for the same

**Comment by iamareebjamal** [comment]
Please don't do this. PRs should not be closed. You can update the branch and PR will be updated accordingly

**Comment by sridharjajoo** [comment]
Ok , I'll take care about this

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+email, or any other method with the maintainers of this repository before making a change. 
+
+Please note we have a [Code Of Conduct](https://github.com/fossasia/open-event-orga-app/blob/development/CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## Issues Process
+
+1. First step while searching for issues is to check the ones with `core` and `high-priority` labels.
+2. Then `unimplemented` issues should be checked which got closed due to lack of attention but still need to be implemented.
+3. Then `deferred` issues may be valid now to take up.
+4. If no such criteria meets, move on to [create new issues](https://github.com/fossasia/open-event-orga-app/issues/new) during an issue hunt. Of course issues about serious bugs and new ideas about features are always welcome.
```
Comment (iamareebjamal) [inline CONTRIBUTING.md]: No, link to the issue buttons in README

**Comment by sridharjajoo** [comment]
@iamareebjamal I have added the links to 3 issue buttons as in README.md . Is it correct?

**Pull request merged**